table/command.go

```diff
@@ -102,7 +101,11 @@ func (o Options) Run() error {
 	}
 
 	m := tm.(model)
-	fmt.Println(strings.Join([]string(m.selected), o.Separator))
+
+	w := csv.NewWriter(os.Stdout)
+	w.Comma = reader.Comma
```

Comment:
Even though `reader.Comma` will contain the right value, I would probably still set this to `o.Separator`. The code above might change in the future and suddenly `reader.Comma` isn't initialized correctly.

Reply:
f6d23cd